src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -27,10 +27,12 @@
 #include <openrct2/world/SmallScenery.h>
 
 static constexpr const rct_string_id WINDOW_TITLE = STR_NONE;
-constexpr int32_t WINDOW_SCENERY_WIDTH = 634;
+constexpr int32_t WINDOW_SCENERY_MIN_WIDTH = 634;
 constexpr int32_t WINDOW_SCENERY_HEIGHT = 180;
+constexpr int32_t SCENERY_TAB_WIDTH = 31;
 constexpr int32_t SCENERY_BUTTON_WIDTH = 66;
 constexpr int32_t SCENERY_BUTTON_HEIGHT = 80;
+constexpr int32_t MAX_TABS = 32;
```

Comment:
You want to avoid this leaking out of the file.

```suggestion
static constexpr int32_t MAX_TABS = 32;
```